starknet_py/net/schemas/gateway.py

```diff
@@ -217,6 +217,7 @@ class StarknetBlockSchema(Schema):
     block_hash = Felt(data_key="block_hash", required=True)
     parent_block_hash = Felt(data_key="parent_block_hash", required=True)
     block_number = fields.Integer(data_key="block_number", required=True)
+    sequencer_address = Felt(data_key="sequencer_address", required=True)
```

Comment:
Let's get more info on this, is this returned on testnet?

Reply:
It seems that after running this code:
```python
    client = GatewayClient(net="testnet")
    res = await client.get_block(block_number="latest")
 ```
`res` in debugger looks like this:


![image](https://github.com/software-mansion/starknet.py/assets/23462504/76997933-584e-4750-ae47-eaae2c1cc7d1)

Reply:
So i think the answer is yes